.env.development

```diff
@@ -1,3 +1,3 @@
-API_SERVICE_BACKEND_URL_OPTIONS={"uk":"http://localhost:3000","xi":"http://localhost:3000" }
+API_SERVICE_BACKEND_URL_OPTIONS={"uk":"https://dev.trade-tariff.service.gov.uk","xi":"https://dev.trade-tariff.service.gov.uk/xi"}
```

Comment:
This can be 

```ruby
API_SERVICE_BACKEND_URL_OPTIONS={uk:"https://dev.trade-tariff.service.gov.uk", xi:"https://dev.trade-tariff.service.gov.uk/xi"}
```
Then you don't need to symbolize this later in the code where you use it.

Reply:
```
json = '{uk:"https://dev.trade-tariff.service.gov.uk", xi:"https://dev.trade-tariff.service.gov.uk/xi"}'
=> "{uk:\"https://dev.trade-tariff.service.gov.uk\", xi:\"https://dev.trade-tariff.service.gov.uk/xi\"}"
[8] pry(main)> JSON.parse(json)
JSON::ParserError: 783: unexpected token at '{uk:"https://dev.trade-tariff.service.gov.uk", xi:"https://dev.trade-tariff.service.gov.uk/xi"}'
from /home/william/.asdf/installs/ruby/2.7.0/lib/ruby/2.7.0/json/common.rb:156:in `parse'
```

Reply:
Ah interesting. Fair!